scripts/linters/general_purpose_linter.py

```diff
@@ -142,15 +142,16 @@
             .EXCLUDED_BYPASS_SECURITY_TRUST_DIRECTORIES)
     },
     {
-        'regexp': re.compile(r'\b(ddescribe|fdescribe)\('),
+        'regexp': re.compile(r'\b(ddescribe|fdescribe|xdescribe)\('),
```

Comment:
Do not implement it here, use custom eslint checks to implement this check.

Reply:
@Hudda Do you want me to separate `ddescribe`, `fdescribe` and `xdescribe` in the same file right?
I will be adding a new rule in the `BAD_PATTERNS_JS_AND_TS_REGEXP` right?
please mention otherwise.

Reply:
@Radesh-kumar You need to implement this check in this directory https://github.com/oppia/oppia/tree/develop/scripts/linters/custom_eslint_checks/rules